specs/agents/logging.md

```diff
@@ -41,6 +41,8 @@ Also, the documentation should not mention the old log levels going forward.
 
 ## Logging Preamble
 
+For supportability reasons, all agents MUST print this preamble on startup.
```

Comment:
I wonder if this "MUST" will cause confusion. For the CLI use case I mentioned earlier where the user of APM may explicitly want to avoid the logging preamble output, we had discussed telling them to set `logging_level: 'WARN'` to avoid the output. Can this be a "SHOULD"?

Reply:
Maybe specify that agents MUST log this preamble on an info log level?

Reply:
@trentm I don't think SHOULD is correct here, as we expect agents to implement this in any case. Only the log level decides whether that output should be visible or not.

@felixbarny the next line already states that.

However, I can try to combine these 2 sentences to make it hopefully less confusing.